docs/LICENSE.txt

```diff
@@ -1,12 +0,0 @@
-Accessibility Insights for Web
```

Comment:
Remove this file and docs/NOTICE.txt from ignore lists in .prettierignore and copyright-header.config.json?